### DISCUSSION TITLE
Question about .bind_visibility_from() when using ui.select()

### DISCUSSION THREAD

**Discussion by miccs0** [zauberzeug/nicegui#3049] [Q&A] [answered]
### Question

The thing is that I would like to only show the "comments" element when my "data" ui.select() option is "1", when it is "2" I would like the "comments" ui.input() to be invisible.
I am aware this is a simple question, but I can't seem to find an answer to it!
Thanks a lot.

```py
data = ui.select(label='Status', options=[1, 2])
comments = ui.input('Comments').bind_visibility_from(data, 'value')
```

**Comment by falkoschindler** [comment, marked as answer]
Hi @miccs0,

You can specify the value for when the element should be visible like this:

```py
ui.input('Comments').bind_visibility_from(data, 'value', value=1)
```

This is a shorthand for explicitly defining the conversion from an arbitrary value to a boolean visibility like this:
```py
ui.input('Comments').bind_visibility_from(data, 'value', lambda v: v == 1)
```

**Reply by miccs0** [reply]
Thanks again for all your help! @falkoschindler
